### PR TITLE
chore: Updates in the core package Part 6

### PR DESCRIPTION
**Pull request by gbockus-sf**
### What does this PR do?
Files touched in the core package

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/auth/authParamsGatherer.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   CancelResponse,
```
Comment (gbockus-sf) [inline packages/salesforcedx-vscode-core/src/commands/auth/authParamsGatherer.ts:11]: This PR is big, but mostly just prettier and module name changes.

File: packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts [hunk 2]
```diff
@@ -18,11 +18,11 @@ import {
   SfdxWorkspaceChecker
 } from '../util';
 
-import { getRootWorkspacePath } from '../../util';
+import { workspaceUtils } from '../../util';
 
 import { ConfigFile } from '@salesforce/core';
-import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode/out/src/helpers';
-import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode';
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts:24]: `isNullOrUndefined` seems generic and not vs-specific.  Can it be moved from salesforcedx-utils-vscode to salesforcedx-utils?

Comment (jeffb-sfdc) [reply packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts:24]: maybe address in a follow up ticket?

File: packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts [hunk 3]
```diff
@@ -93,7 +99,10 @@ export class GetComponentName
   }
 }
 
-export async function inputGuard(sourceFsPath: string, newName: string): Promise<string> {
+export async function inputGuard(
+  sourceFsPath: string,
+  newName: string
+): Promise<string> {
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts:105]: I noticed something here.  Is `newName` a string, or is it a `Promise<string>`?  Seems like it's just a string, and `inputGuard` should be changed to:
```
export async function inputGuard(
  sourceFsPath: string,
  newName: string
): string {
```

Comment (jeffb-sfdc) [reply packages/salesforcedx-vscode-core/src/commands/forceRenameLightningComponent.ts:105]: nm...

File: packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts [hunk 4]
```diff
@@ -39,18 +34,23 @@ export abstract class BaseTemplateCommand extends SfdxCommandletExecutor<
...
     execution.processExitSubject.subscribe(async data => {
       this.logMetric(execution.command.logName, startTime, {
         dirType: this.identifyDirType(response.data.outputdir)
       });
-      if (data !== undefined && String(data) === '0' && hasRootWorkspace()) {
-        const outputFile = this.getPathToSource(response.data.outputdir, response.data.fileName);
-        const document = await vscode.workspace.openTextDocument(
-          outputFile
+      if (
+        data !== undefined &&
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts:45]: how about changing this from `data !== undefined &&` to `data &&`

File: packages/salesforcedx-vscode-core/src/commands/util/preconditionCheckers.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode/out/src/helpers';
+import { isNullOrUndefined } from '@salesforce/salesforcedx-utils-vscode';
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-core/src/commands/util/preconditionCheckers.ts:7]: same comment - isNullOrUndefined seems generic and should be in salesforcedx-utils

File: packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts [hunk 6]
```diff
@@ -10,10 +10,14 @@ import * as os from 'os';
...
 import { nls } from '../messages';
-import { getRootWorkspacePath, hasRootWorkspace } from '../util';
+import { workspaceUtils } from '../util';
 
-const CONFIG_FILE = hasRootWorkspace()
-  ? path.join(getRootWorkspacePath(), '.sfdx', 'sfdx-config.json')
+const CONFIG_FILE = workspaceUtils.hasRootWorkspace()
+  ? path.join(
+      workspaceUtils.getRootWorkspacePath(),
+      '.sfdx',
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts:18]: Is it OK to use a path with `.sfdx` here, or is this another case where we need to call an API?

File: packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts [hunk 7]
```diff
@@ -43,7 +43,7 @@ export class ComponentUtils {
       folderName ? `${metadataType}_${folderName}` : metadataType
     }.json`;
     const componentsPath = path.join(
-      getRootWorkspacePath(),
+      workspaceUtils.getRootWorkspacePath(),
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts:46]: same comment as before... is it OK to use a path with `.sfdx` here, or is there an API we should use?

Comment (jeffb-sfdc) [reply packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts:46]: If we stick with sfdx, then to match the other code this should be added:
`import { SFDX_PROJECT_FILE } from '../constants';`

File: packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts (hunk 6 above)
Comment (jeffb-sfdc) [reply packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts:18]: If we stick with sfdx, then to match the other code this should be added:
`import { SFDX_PROJECT_FILE } from '../constants';`

File: packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts (hunk 2 above)
Comment (gbockus-sf) [reply packages/salesforcedx-vscode-core/src/commands/auth/forceAuthDevHub.ts:24]: 100% agree. Will add it to my list of follow ups

File: packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts (hunk 6 above)
Comment (gbockus-sf) [reply packages/salesforcedx-vscode-core/src/decorators/scratch-org-decorator.ts:18]: yup, ok for now.  We have a bunch of work items to address all these in the core v3 epic.

**Review by jeffb-sfdc — APPROVED** [review]

**Pull request closed without merging**